Update README.md (#1)
chanzuckerberg/napari-hub: README.md

# napari hub

> This repository contains the source code for the _napari hub_, a place to find, evaluate, and install _napari_ plugins, build by the Imaging team at the Chan Zuckerberg Initiative.

Reviewer: ```suggestion
This repository contains the source code for the _napari hub_, a place to find, evaluate, and install _napari_ plugins, built by the Imaging team at the Chan Zuckerberg Initiative.
```

> This repository contains the source code for the _napari hub_, a place to find, evaluate, and install _napari_ plugins, build by the Imaging team at the Chan Zuckerberg Initiative.

Reviewer 2: ```suggestion
This repository contains the source code for the _napari hub_: a place to find, evaluate, and install _napari_ plugins, built by the Imaging team at the Chan Zuckerberg Initiative.
```

> This repository contains the source code for the _napari hub_, a place to find, evaluate, and install _napari_ plugins, built by the Imaging Team at the Chan Zuckerberg Initiative.

Reviewer 3: Should we link to our Imaging page on the CZI website? I'll leave the decision up to you if it makes sense. https://chanzuckerberg.com/science/programs-resources/imaging/

## What we're building now

For our first _alpha/v0_ release in June 2021, the _napari hub_ will focus on making it easy for [Research Biologists, Imaging Scientists, and Bioimage Analysts](https://cziscience.medium.com/user-experience-research-in-the-imaging-field-6bb89e592bb9) to find, evaluate, and install _napari_ plugins distributed through PyPI.

> Through talking with users about challenges and needs finding analysis solutions of bioimaging problems, we focused on a handful of critical features for our first release, including...

Reviewer 2: ```suggestion
After talking with users about the challenges and needs finding analysis solutions to bioimaging problems, we focused on a handful of critical features for our first release, including...
```

Reviewer 2: I like the "through" collaborative intent but I'm not sure the grammar holds up on the second part of the sentence.

> After talking with users about the challenges and needs finding analysis solutions to bioimaging problems, we focused on a handful of critical features for our first release, including...

Reviewer 3: Minor: Added "in"
After talking with users about the challenges and needs **in** finding analysis solutions to bioimaging problems, we focused on a handful of critical features for our first release, including...

> - *Global Search Bar* which returns results from multiple metadata sources (title, author, summary, description, etc)

Reviewer 2: ```suggestion
- *Global Search Bar* which returns results from multiple metadata sources (title, author, summary, description, etc.)
```

Reviewer 2: also, I think all of these bullet points should be lower case? "global search bar"

> - *Global Search Bar* which returns results from multiple metadata sources (title, author, summary, description, etc)

Reviewer 2: ```suggestion
- *global search bar* which returns results from multiple metadata sources (title, author, summary, description, etc)
```

> - *napari-specific metadata*, such as author(s), links to external docs, support channels, etc, that plugin developers can easily define in PyPI metadata or Github config files

Reviewer 2: ```suggestion
- *napari-specific metadata*, such as author(s), links to external docs, support channels, etc., that plugin developers can easily define in PyPI metadata or Github config files
```

> - *napari-specific description* Must have: supports both napari-specific readme or default to regular readme if plugin developer does not use template, ability to pull from developer’s manual input readme in github to populate individual plugin page in discovery site.

Reviewer 3: I would remove "Must have:" to be consistent

Author: woops, that was a copy-paste error

> - *Install “Call To Action”* button for each plugin with plugin-specific instructions on how to install

Reviewer 2: ```suggestion
- *install “call-to-action”* button for each plugin with plugin-specific instructions on how to install
```

> - *Filter/Sort* available plugins based on limited selection of metadata (2-3 filters, 1-2 sorts) that can be handled client side

Reviewer 2: ```suggestion
- *filter/sort* available plugins based on limited selection of metadata (2-3 filters, 1-2 sorts) that can be handled client side
```

> - *Mailing List* sign up for announcements and updates for the _napari hub_

Reviewer 2: ```suggestion
- *mailing list* sign up for announcements and updates for the _napari hub_
```

## What we're building next

> After our _alpha_ launch, we're very excited to keep iterating and building new features that are of value to the bioimaging community, supporting current and future _napari_ users and developers.

Reviewer: ```suggestion
After our _alpha_ launch, we're very excited to keep iterating and building new features that are of value to the imaging community, supporting current and future _napari_ users and developers.
```

> We have a host of ideas for ways that we can help biologists learn if a plugin is right for them, for imaging scientists to share their knowledge with core facility users and collaborators, and ways to help lower barriers for bioimage analysts in creating, maintaining, and supporting their plugins.

Reviewer: ```suggestion
We have a host of ideas for ways that we can help scientists learn if a plugin is right for them, for imaging scientists to share their knowledge with core facility users and collaborators, and ways to help lower barriers for image analysts in creating, maintaining, and supporting their plugins.
```

When we have a clearer idea of our roadmap beyond our June release, we'll share it here. In the meantime, feel free to share your ideas by submitting a [feature request](https://github.com/chanzuckerberg/napari-hub/issues)!

## About

The _napari hub_ is a service of the Chan Zuckerberg Initiative.

> _napari_ is a consensus-based community project and an open source tool that enables high performance visualization and exploration of a broad range of imaging data, including microscopy, medical imaging, geospatial data, and more, with clearly defined governance model, mission, and values.

Reviewer 3: We should probably have at least one link out to napari.org or napari GH. Maybe here?

At CZI, we support the development of open source and community run tools like _napari_, as part of our mission to accelerate biomedical research and help every scientist make progress faster.

> The CZI Imaging Team seeks to remove barriers in the analysis of imaging and microscopy data and make it easier for biologists to access emerging methods for bioimage analysis that leverage machine learning.

Reviewer 2: do we have consistent choice for capitals for team name? "Imaging Team" vs "Imaging team" (both are present here)

Author: I have a mild preference for all caps. Just tweaked to make it consistent here.

CZI sees promise in _napari_ and seeks to assist its development by providing resources not always available to the open source community, including dedicated user research, design, and engineering support.
CZI is proud to collaborate with the science community to accelerate research and enable open science for all.

> ### Team

Reviewer 2: is there an order here? if not i'd suggest A-Z by last name?

Author: this was A-Z by first name, but yes, last name is prob better

> - Jeremy Asuncian, Engineering

Reviewer: I'd add github handles, i think it's quite nice, makes it a little more clear who people are

Author: done

> - Jeremy Asuncian, Engineering ([@codemonkey800](https://github.com/codemonkey800))

Author: Thank you :D

- Kira Evans, Engineering ([@kne42](https://github.com/kne42))
- Justin Kiggins, Product ([@neuromusic](https://github.com/neuromusic))
- Justine Larsen, Engineering ([@justinelarsen](https://github.com/justinelarsen))
- Ziyang Liu, Engineering ([@ziyangczi](https://github.com/ziyangczi))
- Lucy Obus, User Experience ([@LCObus](https://github.com/LCObus))

> - Lia Prins, Design ([@liaprins-czi](https://github.com/liaprins-czi))

Reviewer: I think would be nice to add me, as I am helping, at least i hope ;-)
```suggestion
- Lia Prins, Design ([@liaprins-czi](https://github.com/liaprins-czi))
- Nicholas Sofroniew, Product ([@sofroniewn](https://github.com/sofroniewn))
```

Author: of course :D
